docs/WindowsBuild.md

```diff
@@ -41,17 +43,18 @@ Windows.
 
 ### 2. Clone the repositories
 1. Create a folder to contain all the Swift repositories
-1. Clone `apple/swift-cmark` into a folder named `cmark`
-1. Clone `apple/swift-clang` into a folder named `clang`
-1. Clone `apple/swift-llvm` into a folder named `llvm`
+1. Clone `apple/swift-cmark` into a folder named `swift-cmark`
+1. Clone `apple/swift-clang` into a folder named `swift-clang`
+1. Clone `apple/swift-llvm` into a folder named `swift-llvm`
+1. Clone `apple/swift-compiler-rt` into a folder named `swift-compiler-rt`
```

Comment:
Why are you changing the names here?  This layout is similar to what is used on Linux and macOS.

Reply:
Because these are the folder names that we get when we do a git clone with the repository URL as published by GitHub:

```bash
$ git clone https://github.com/apple/swift-cmark.git
Cloning into 'swift-cmark'...
remote: Counting objects: 9101, done.
remote: Compressing objects: 100% (14/14), done.
remote: Total 9101 (delta 13), reused 1 (delta 0), pack-reused 9087
Receiving objects: 100% (9101/9101), 2.79 MiB | 3.71 MiB/s, done.
Resolving deltas: 100% (6524/6524), done.
$ ls
swift-cmark
```

Reply:
Sure, but the expectation is that the repositories will be renamed.  See https://github.com/apple/swift/blob/master/utils/update-checkout-config.json